locale/circuitpython.pot

```diff
@@ -2968,6 +2968,10 @@ msgstr ""
 msgid "destination buffer must be an array of type 'H' for bit_depth = 16"
 msgstr ""
 
+#: shared-bindings/usb/cdc_host/__init__.c
+msgid "device must be a usb.core.Device object"
```

Comment:
Instead of adding a new message, reuse `"%q must be of type %q, not %q"`. (I'd search for where it is used to copy it's example.)